packages/teams-js/src/internal/utils.ts

```diff
@@ -402,30 +402,30 @@ export function inServerSideRenderingEnvironment(): boolean {
 }
 
 /**
- * @param appID The app ID to validate
- * @throws Error if appID is not a valid GUID
+ * @param Id The Id to validate
```

Comment:
Can you also add documentation for the error property (which I suggest we call `errorToThrow`) to indicate that if passed in, the method will throw errorToThrow, otherwise will throw a default error.